Settings.ui

```diff
@@ -3235,6 +3235,69 @@
             </child>
           </object>
         </child>
+        <child>
+          <object class="GtkListBoxRow" id="listboxrow_scroll_panel_options_show_ws_popup">
+            <property name="width_request">100</property>
+            <property name="height_request">80</property>
```

Comment:
Please remove the size requests so the row has the same size as the one above it:

![image](https://user-images.githubusercontent.com/33044886/84607205-00bf4680-ae7a-11ea-93bd-f91d7cfb51ca.png)

Reply:
:heavy_check_mark: done